bookclub-frontend/src/components/bookclub/BookInfo.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container'
```

Comment:
In order to concisely pull from all four components from bootstrap, can you change it to show: 
`import { Container, Row, Col, Image } from 'react-boostrap' `? 
This will add a level of readability and removes repetition

Reply:
i definitely can do that. i got this from the docs:
"You should import individual components like: react-bootstrap/Button rather than the entire library. Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client."
import Button from 'react-bootstrap/Button';
// or less ideally
import { Button } from 'react-bootstrap';

It would look much cleaner to de-structure it that way, however i would be importing the whole library into that file for just the four components. If you prefer the cleaner syntax, i will most certainly fix it.

Reply:
I saw that documentation as well! I think when they say "rather than the entire library", they're referring to importing all of `react-bootstrap`, instead of specifically `react-bootstrap/Card`. 
The reason why the `import { Button } from 'react-bootstrap` was called "less ideally" was because that would be less ideal if you only used _one_ component from react-bootstrap, but since you're using four of them, it's considerably more ideal (less characters) and cleaner.